cmd/simRecon/simRecon_test.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"github.com/vertgenlab/gonomics/fasta"
+	"github.com/vertgenlab/gonomics/fileio"
+	"strings"
+	"testing"
+)
+
+func accEqual(a string, b string) bool {
```

Comment:
I am not sure the logic of this function is what you are looking for.  If I am reading it correctly, it takes two file names and if there is at least one line in file A, for which there is also a line in file B that has the first two columns match then we return true.
I am wondering if you might be looking for all the lines to match, but want to allow a different ordering of the lines?

Reply:
Yeah I got hung up on checking all the names first but now that you point it out it isn't totally necessary. Instead of splitting on the tab should I just leave the line as a single "word" and check them against each other?